src/WebJobs.Extensions.DocumentDB/DocumentDBAttribute.cs

```diff
@@ -64,7 +64,7 @@ public DocumentDBAttribute(string databaseName, string collectionName)
         /// Optional. A string value indicating the app setting to use as the DocumentDB connection string, if different
         /// than the one specified in the <see cref="DocumentDBConfiguration"/>.
         /// </summary>
-        public string ConnectionString { get; set; }
+        public string ConnectionStringSetting { get; set; }
```

Comment:
The value on DocuemntDBConfiguration is an actual connection string, or is it a setting as well? Same question for notification hubs